### PR TITLE
5454 Remove service_regions

### PR DESCRIPTION
**Pull request by alexwllms**
@benlovell @lshepstone

### CODE REVIEW

File: app/views/questionnaires/new.html.erb [hunk 1]
```diff
@@ -41,63 +41,46 @@
...
-                <%= f.label :address_county, t('questionnaire.address.label_four'), class: 'form__label-heading' %>
-                <%= f.text_field :address_county %>
-            <% end %>
-
-            <%= f.form_row :address_postcode do %>
-                <%= f.errors_for :address_postcode %>
-                <%= f.label :address_postcode, t('questionnaire.address.label_five'), class: 'form__label-heading' %>
-                <%= f.text_field :address_postcode %>
-            <% end %>
+        <fieldset class="form__group">
```
Comment (benlovell) [inline app/views/questionnaires/new.html.erb]: Weird that this hunk became part of the diff...

**Comment by benlovell** [comment]
You missed a few things. You'll need to remove the `ServiceRegion` model, specs and anything left over.

File: app/views/questionnaires/new.html.erb (hunk 1 above)
Comment (alexwllms) [inline app/views/questionnaires/new.html.erb]: I think that's because I moved a </section> tag and that section got indented. My bad.

**Comment by benlovell** [comment]
Nice one. Squash these commits and get to merging :hammer: :ship:

**Comment by alexwllms** [comment]
tumblr_m8rop9rho01qhk6l1

**Comment by lshepstone** [comment]
:+1:

**Pull request merged**